src/openrct2/network/NetworkAction.cpp

```diff
@@ -223,7 +223,8 @@ const std::vector<NetworkAction> NetworkActions::Actions =
     }, {
         STR_ACTION_EDIT_SCENARIO_OPTIONS, "PERMISSION_EDIT_SCENARIO_OPTIONS",
         {
-            GAME_COMMAND_EDIT_SCENARIO_OPTIONS
+            GAME_COMMAND_EDIT_SCENARIO_OPTIONS,
+            GAME_COMMAND_PLACE_PEEP_SPAWN
```

Comment:
I think scenario options refer to stuff like the goal, pay-per-ride settings, etc. I would put peep spawn permissions in the same category as placing park entrances.

Reply:
I originally thought about doing it that way, but realized the potential for easier abuse. There are less chances of someone getting scenario options permissions than park properties. it's harder to catch abuse on this thing in specific, too, whereas you can easily see someone renaming a park, changing the park price, or moving the park entrance somewhere.  if you trust someone with having the option to modify the pay-per-ride setting, you'd likely trust someone with the ability to change the peep spawn.

Reply:
I agree with wolfreak here. Looking at what permissiosn they grant exactly, I think it fits better:`PERMISSION_PARK_PROPERTIES`, while `PERMISSION_EDIT_SCENARIO_OPTIONS`

Reply:
I agree with wolfreak here. Looking at what permissiosn they grant exactly, I think it fits better: `PERMISSION_PARK_PROPERTIES`
- 
- 

`PERMISSION_EDIT_SCENARIO_OPTIONS`
- Set money
- Forbit marketing
- Guests preferences

Reply:
I agree with wolfreak here. Looking at what permissions they grant exactly, I think it fits better.
`PERMISSION_PARK_PROPERTIES` allows changing settings that you can change from the park window (buying land rights, changing park name, set entrance fee), nothing guests related.
`PERMISSION_EDIT_SCENARIO_OPTIONS` allows changing settings that already have to do with guests generation (spawn rate, initial money, initial happiness).

I don't expect changing guest entry point to really be a abusive issue. Players that have yet to be trusted by a server owner often have neither of those permissions.

I can imagine it being confusing to know which group will grant this permission however. @Gymnasiast If it's possible to list them under both categories, do you think that's too inconsistent or causes too much confusion? If not, that might be nice to do here.

Reply:
> Perhaps a sandbox permission group may be better for both of these actions instead.

That sounds better.

Reply:
Would this be able to go into a separate pr? Or would someone be able to modify the pr and help add it? Otherwise I'm not sure when I'll be able to get around to doing it. It may have to wait a few days.

Reply:
In that case, put it under PERMISSIONS_PARK_PROPERTIES.